vision_datasets/commands/converter_IrisCoco_to_AMLCoco.py

```diff
@@ -0,0 +1,96 @@
+""" 
```

Comment:
can you run flake8 to check if any warning exists?

Reply:
Done. Passing now